editor/MapNodeEditor.tsx

```diff
@@ -10,8 +10,7 @@ import { getComponent } from '@etherealengine/engine/src/ecs/functions/Component
 
 export const MapNodeEditor: EditorComponentType = (props) => {
   const { t } = useTranslation()
-
-  const mapComponent = getComponent(props.node.entity, MapComponent)
+  const mapComponent = getComponent(props.entity, MapComponent)
```

Comment:
this should be useComponent to be reactive, then each value will been .value appended